scheds/rust/scx_p2dq/src/bpf/main.bpf.c

```diff
@@ -909,7 +909,12 @@ static __always_inline int p2dq_running_impl(struct task_struct *p)
 	if (freq_control && taskc->dsq_index == nr_dsqs_per_llc-1) {
 		scx_bpf_cpuperf_set(task_cpu, SCX_CPUPERF_ONE);
 	}
-	taskc->last_run_at = bpf_ktime_get_ns();
+
+	u64 now = bpf_ktime_get_ns();
+	if (taskc->last_run_started == 0)
```

Comment:
I'm wondering, can you just set last_run_started in ops.runnable() instead of doing this check in ops.running()?

Reply:
I tried that as well, it didn't seem to make much of a difference either way.